src/World1/world1/plots.jl

```diff
@@ -0,0 +1,359 @@
+        (w1.qlm,  0,     8,     "Quality of life from materials"),
+        (w1.ql,   0,     40,    "Quality of life"),
+        (w1.ciaf, 0.25,  0.65,  "Capital in agriculture"),
+        (w1.nr,   700e9, 900e9, "Natural resources"),
+    ]
+
+    return variables
+end
+
+function _variables_10()
+    @named w1 = world1()
+
+    variables = [
+        (w1.pop,  0,     4e9,   "Population"),
+        (w1.polr, 0,     12,    "Pollution"),
+        (w1.ci,   0,     4e9,   "Capital investment"),
+        (w1.fr,   0,     2,     "Food ratio"),
+        (w1.cr,   0,     0.8,   "Crowding ratio"),
+        (w1.msl,  0,     8,     "Material standard of living"),
+        (w1.qlm,  0,     8,     "Quality of life from materials"),
+        (w1.ql,   0,     20,    "Quality of life"),
+        (w1.ciaf, 0.25,  0.65,  "Capital in agriculture"),
+        (w1.nr,   500e9, 900e9, "Natural resources"),
+    ]
+
+    return variables
+end
+
+
+fig_std(; kwargs...) = plotvariables(standardrunsolution(), (t, 1900, 2100), _variables_std(); title="WORLD1-STD", showaxis=false, showlegend=true, kwargs...)
+
+function fig_1(; kwargs...)
+    new_params = copy(_params)
+    new_params[:cigc] = 0.5
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_1(); title="W1-7/5-1", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_2(; kwargs...)
+    new_params = copy(_params)
+    new_params[:cigc] = 0.5
+
+    new_inits = copy(_inits)
+    new_inits[:ciaf] = 0.25
+
+    sol = solve(standard_run(inits=new_inits, params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_2(); title="W1-7/5-2", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_3(; kwargs...)
+    new_params = copy(_params)
+    new_params[:cigc] = 0.5
+
+    new_inits = copy(_inits)
+    new_inits[:ciaf] = 0.25
+
+    sol = solve(standard_run(inits=new_inits, params=new_params), (1900, 2400))
+
+    plotvariables(sol, (t, 1900, 2400), _variables_3(); title="W1-7/5-3", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_4(; kwargs...)
+    new_params = copy(_params)
+    new_params[:brn] = 0.025
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_4(); title="W1-7/5-4", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_5(; kwargs...)
+    new_params = copy(_params)
+    new_params[:brn] = 0.035
+
+    new_inits = copy(_inits)
+    new_inits[:ciaf] = 0.3
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
+
+    plotvariables(sol, (t, 1900, 2100), _variables_5(); title="W1-7/5-5", showaxis=false, showlegend=true, kwargs...)
+end
+
+function fig_6(; kwargs...)
+    new_params = copy(_params)
+    new_params[:brn] = 0.032
+
+    new_inits = copy(_inits)
+    new_inits[:ciaf] = 0.3
+
+    sol = solve(standard_run(params=new_params), (1900, 2100))
```

Comment:
The initializations were missing from the arguments:
```suggestion
    sol = solve(standard_run(inits=new_inits, params=new_params), (1900, 2100))
```
After the above fix, output seems equal to the figure in the book.